.github/workflows/package-build.yml

```diff
@@ -66,18 +55,83 @@ jobs:
           asset_path: ${{ env.UPLOAD_FILE }}
           asset_name: ${{ env.UPLOAD_FILE }}
           asset_content_type: ${{ env.UPLOAD_ASSET_TYPE }}
+  build-windows:
+    runs-on: windows-2019
+    strategy:
+      fail-fast: false
+    steps:
+    - uses: actions/checkout@v3
+    #- name: msys setup
```

Comment:
Any particular reason to keep this? I suppose since we have native builds, we don't really need MSYS

Reply:
Not quite. Using the mingw or cygwin based toolchain isn't the same as using msys as unixy scripting environment are two separate things.

The main script which orchestrates the build process is a makefile. So MSYS does get used. Although with current setup if a need for additional setup does rise, it can probably be done build_win.sh. Will remove this in next cleanup commit.

Reply:
Removed commented out block.